install/get-tor-windows.py

```diff
@@ -32,21 +32,23 @@
 
 def main():
     exe_url = (
-        "https://archive.torproject.org/tor-package-archive"
-        "/torbrowser/8.0.5/torbrowser-install-8.0.5_en-US.exe"
+        "https://archive.torproject.org/tor-package-archive/torbrowser/9.0.7/torbrowser-install-9.0.7_en-US.exe"
     )
-    exe_filename = "torbrowser-install-8.0.5_en-US.exe"
-    expected_exe_sha256 = (
-        "860fdd06e4ea8dd4c46f221676251f5c" "c528676d4e256559ee3831a5f97492f1"
+    asc_url = (
+        "https://archive.torproject.org/tor-package-archive/torbrowser/9.0.7/torbrowser-install-9.0.7_en-US.exe.asc"
     )
+    exe_filename = "torbrowser-install-9.0.7_en-US.exe"
+
+
     # Build paths
     root_path = os.path.dirname(
         os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
     )
     working_path = os.path.join(os.path.join(root_path, "build"), "tor")
     exe_path = os.path.join(working_path, exe_filename)
+
     dist_path = os.path.join(
-        os.path.join(os.path.join(root_path, "dist"), "onionshare"), "tor"
+        os.path.join(os.path.join(root_path, "dist"), "hyperdome"), "tor"
```

Comment:
Pathlib would make this a lot cleaner. It can be a future thing tho

Reply:
yep, part of me wanting to re-write this whole module (and also combine the macos and windows ones since there's only a few lines difference)